INSTALL

```diff
@@ -24,14 +25,10 @@ Table of contents:
 
 The following packages must be installed to use Beluga:
 
-ocaml 3.12.2+:           http://caml.inria.fr/download.en.html
+ocaml 4.07.1+:           http://caml.inria.fr/download.en.html
 extlib:                  http://code.google.com/p/ocaml-extlib/downloads/list
 ulex:                    http://www.cduce.org/download.html#side
 
-Users not running Windows may wish to build using omake:
-
-omake:			 http://omake.metaprl.org/index.html
```

Comment:
I installed omake but when I ran it, it complained that it couldn't find any omake files; I looked at the history and found that the project is no longer using omake, so this part of the installation instructions was obsolete and misleading.